admin/common_auth/forms.py

```diff
@@ -0,0 +1,5 @@
+from django import forms
+
+class LoginForm(forms.Form):
+    username = forms.CharField(label=(u'User Name'), required=True)
```

Comment:
It may or may not become confusing later that the form refers to this field as username and the models and manager all refer to it as email.